src/roswire/ros1/source.py

```diff
@@ -57,9 +57,41 @@ def get_cmake_info(
                 target.entrypoint = entrypoint
         return info
 
+    def _find_package_devel_workspaces(self, package: Package) -> t.Set[str]:
```

Comment:
The name and docs of the method don't quite make sense any more. Suggestion: keep the original `_find_package_workspace` function from ROSDiscover and move the finding of include directories to `package_paths`.